src/hotspot/share/gc/shared/workerDataArray.inline.hpp

```diff
@@ -158,39 +156,31 @@ void WorkerDataArray<T>::set_all(T value) {
 
 template <class T>
 void WorkerDataArray<T>::print_summary_on(outputStream* out, bool print_sum) const {
-  if (_is_serial) {
-    out->print("%s:", title());
-  } else {
-    out->print("%-30s", title());
-  }
+  out->print("%-30s", title());
 
   uint start = 0;
   while (start < _length && get(start) == uninitialized()) {
     start++;
   }
   if (start < _length) {
-    if (_is_serial) {
-      WDAPrinter::summary(out, get(0));
```

Comment:
From what I can tell these two function will no longer be used after your change:
```
WDAPrinter::summary(outputStream* out, double time);
WDAPrinter::summary(outputStream* out, size_t value);
```
Please remove them.

Reply:
Removed.